dataprofiler/profilers/json_decoder.py

```diff
@@ -0,0 +1,68 @@
+"""Contains methods to decode components of a Profiler."""
+
+import json
+
+from .base_column_profilers import BaseColumnProfiler
+from .categorical_column_profile import CategoricalColumn
+
+
+def get_column_profiler_class(class_name: str) -> BaseColumnProfiler:
+    """
+    Use name of class to return default-constructed version of that class.
+
+    Raises ValueError if class_name is not name of a subclass of
+        BaseColumnProfiler.
+
+    :param class_name: name of BaseColumnProfiler subclass retreived by
+        calling type(instance).__name__
+    :type class_name: str representing name of class
+    :return: subclass of BaseColumnProfiler object
+    """
+    profiles = {CategoricalColumn.__name__: CategoricalColumn}
+
+    profile_class = profiles.get(class_name)
+    if profile_class is None:
+        raise ValueError(f"Invalid profiler class {class_name} " f"failed to load.")
+    else:
+        return profile_class(None)
```

Comment:
technically no need for the else as it will error otherwise.
```python
if profile_class is None:
        raise ValueError(f"Invalid profiler class {class_name} " f"failed to load.")
return profile_class(None)
```